jenkins/shared-library/resources/artifacts/scripts/pypi-publish.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -ex
+
+# prepare toml (add requirements)
+while IFS= read -r line; do poetry add "$line"; done < requirements.txt
```

Comment:
this is adding all the requirements to poetry from requirements.txt (this allows us to have one list)

aside: `cat requirements.txt | poetry add` but shellcheck doesn't like that